### ISSUE TITLE
em-http-request CVE causing a TLS warning

### ISSUE THREAD

**Issue by rafdizzle86** [francois2metz/em-eventsource#18]
The [`em-http-request`](https://github.com/igrigorik/em-http-request) gem recently published a new version (1.1.6) which highlights a CVE around a Man in the Middle attack: https://github.com/igrigorik/em-http-request/issues/339. Is it possible to have the gem use TLS  by default?

I am getting this warning when using this gem in various projects:
```
[WARNING; em-http-request] TLS hostname validation is disabled (use 'tls: {verify_peer: true}'), see CVE-2020-13482 and https://github.com/igrigorik/em-http-request/issues/339 for details
```

**Comment by rafdizzle86** [comment]
I think this line perhaps needs to change:

https://github.com/francois2metz/em-eventsource/blob/master/lib/em-eventsource.rb#L196

Add a `tls: {verify_peer: true}` to the `options` argument:

```
    def prepare_request
      conn = EM::HttpRequest.new(@url, {tls: {verify_peer: true}, inactivity_timeout: @inactivity_timeout})
      ...
     end
```

**Comment by francois2metz** [comment]
Hi @rafdizzle86 . Could you please open a pull request for this?

**Comment by francois2metz** [comment]
Fixed and released in the 0.3.2 version! Thanks for the PR.
